docs/api-reference.md

```diff
@@ -608,4 +624,140 @@ end
 As with `setState`, you can set use the constant `Roact.None` to remove a field from the state.
 
 !!! note
-	`getDerivedStateFromProps` is a *static* lifecycle method. It does not have access to `self`, and must be a pure function.
\ No newline at end of file
+	`getDerivedStateFromProps` is a *static* lifecycle method. It does not have access to `self`, and must be a pure function.
+
+---
+
+## ShallowWrapper
+
+### Fields
+
+#### type
+```
+type: {
+	kind: ElementKind
```

Comment:
like with https://github.com/Roblox/roact/issues/227 we might want a mirror and not expose the ElementKind symbols directly, so Elements cannot be impersonated.